view/template/app-user-guide.hbs

```diff
@@ -0,0 +1,286 @@
+<main class="container bear-background">
+    <h1 class="text-xs-center">
+        PROMIS - User Guide For The Mobile App
+    </h1>
+    <h2>
+        Please follow below mentioned steps to download the app on android device and start the survey
+    </h2>
+    <br>
+    <br>
+    <nav>
+        <h2>
+            Index:
+        </h2>
+        <ol>
+            <li>
+                <a href="#step1">
+                    Download the app on mobile device
+                </a>
+            </li>
+            <li>
+                <a href="#step2">
+                    After download, install the app
+                </a>
+            </li>
+            <li>
+                <a href="#step3">
+                    If no notification displayed, go to Downloads to install
+                </a>
+            </li>
+            <li>
+                <a href="#step4">
+                    Installation fails due to unknown source
```

Comment:
I guess I should remove this as we will not be serving the apk anymore from the server machine.